space/space.go

```diff
@@ -302,7 +302,7 @@ func (r *GormRepository) LoadByOwner(ctx context.Context, userID *uuid.UUID, sta
 
 func (r *GormRepository) LoadByOwnerAndName(ctx context.Context, userID *uuid.UUID, spaceName *string) (*Space, error) {
 	res := Space{}
-	tx := r.db.Where("spaces.owner_id=? AND spaces.name=?", *userID, *spaceName).First(&res)
+	tx := r.db.Where("spaces.owner_id=? AND LOWER(spaces.name)=?", *userID, strings.ToLower(*spaceName)).First(&res)
```

Comment:
That is exactly the operation I'd like to avoid because it fetches all space names from the DB and performs the `lower()` operation. @hectorj2f can you please call `EXPLAIN SELECT * FROM spaces WHERE LOWER(name) = 'foo'` to see if the index `spaces_name_idx` is really being utilized?